paasta_tools/cli/cmds/logs.py

```diff
@@ -226,15 +239,15 @@ def check_timestamp_in_range(timestamp, start_time, end_time):
 
 
 def paasta_log_line_passes_filter(
-    line,
-    levels,
-    service,
-    components,
-    clusters,
-    instances,
-    start_time=None,
-    end_time=None,
-):
+    line: str,
+    levels: Sequence[str],
+    service: str,
+    components: Union[str, Iterable[str]],
```

Comment:
Could you please double-check that `componens` is a `Union` and not just `Iterable` here?

We use the `componens`  variable below for
```
parsed_line.get("component") in components
```
Wondering, how all of that is supposed to work if `components` has the `str` type and isn't equal to `"default"`...

Reply:
This particular filtering function will always receive components='default'. Let me give this another pass and clean up this part of the code.